.travis.yml

```diff
@@ -1,13 +1,29 @@
+# by default Travis uses JDK 8u31, which is much too old for Scala 2.12 (versions before 8u102 have issues - https://issues.scala-lang.org/browse/SI-9828)
+# Trusty has 8u131 or later
+dist: trusty
+sudo: required
+
 language: scala
-script: sbt -jvm-opts jvmopts.travis -Dslick.testkit-config=test-dbs/testkit.travis.conf +testAll
+
+scala:
+  - 2.12.3
+  - 2.11.11
+  - 2.10.6
+
+script: sbt ++$TRAVIS_SCALA_VERSION -jvm-opts jvmopts.travis -Dslick.testkit-config=test-dbs/testkit.travis.conf +testAll
```

Comment:
With the current config I think this needs to be `testAll` instead of `+testAll` because you want to build on a specific `$TRAVIS_SCALA_VERSION`. But see my other comment about cross-building within sbt.

Reply:
Fixed